tactics/redexpr.ml

```diff
@@ -25,7 +25,8 @@ open Libobject
 (* call by value normalisation function using the virtual machine *)
 let cbv_vm env sigma c =
   if Coq_config.bytecode_compiler then
-    let ctyp = Retyping.get_type_of env sigma c in
+    (* Hack: drop the evarmap *)
```

Comment:
why?

Reply:
Too annoying to return it, I would have to tweak the contextual functions and whatnot. I just wanted a quick idea of the level of desolation triggered by this nuclear patch.